README.md

```diff
@@ -1,81 +1,47 @@
+
+> **_NOTE:_**  The ICON Bridge is currently in beta and has not completed auditing. Use at your own risk.
+
+# Getting Started
+
+We are currently refactoring our process for getting started and will update this file when it is in a better state.
 
 ### Documents
 
-- [Build Guide](doc/build.md)
-- [Tutorial](doc/tutorial.md)
 - [iconbridge command line](doc/iconbridge_cli.md)
-- [Binance Smart Chain Guide](doc/bsc-guide.md)
-- [Harmony Guide](doc/hmny-guide.md)
+- TODO: add more docs
+
 
 ### Layout
 
 | Directory                            | Description                                                                                       |
 | :----------------------------------- | :------------------------------------------------------------------------------------------------ |
-| /cmd                                 | Root of implement of BMR                                                                          |
-| /cmd/iconbridge                       | Reference implement of BMR. only provide unidirectional relay. (golang)                           |
-| /cmd/iconbridge/relay                 | Implement of common logic of BMR, uses chain package                                              |
-| /cmd/iconbridge/chain                 | BMR module interface, common code, and chain specific packages                                    |
-| /cmd/iconbridge/chain/`<blockchain>`  | Implement of BMR module (`Sender`,`Receiver`), `<blockchain>` is name of blockchain               |
+| /cmd                                 | Root of implement of the relay                                                                          |
+| /cmd/iconbridge/relay                 | Implement of common logic of relay, uses chain package                                              |
+| /cmd/iconbridge/chain                 | Relay module interface, common code, and chain specific packages                                    |
+| /cmd/iconbridge/chain/`<blockchain>`  | Implement of relay module (`Sender`,`Receiver`), `<blockchain>` is name of blockchain               |
 | /common                              | Common code (golang)                                                                              |
 | /doc                                 | Documents                                                                                         |
 | /docker                              | Docker related resources                                                                          |
-| /`<env>`                             | Root of implement of BTP smart contracts, `<env>` is name of smart contract execution environment |
-| /`<env>`/bmc                         | Implement of BMC smart contract                                                                   |
+| /`<env>`                             | Root of implement of ICON Bridge smart contracts, `<env>` is name of smart contract execution environment |
```

Comment:
Perhaps to use particular folder would be more informative. 
For example: jacascore/